README.md

```diff
@@ -1 +1,188 @@
+- [📝 License](#license)
+
+
+# 📖 [Linters Project with Github for HTML and CSS] <a name="about-project"></a>
+
+
+
+**Linters Project with Github will help you detect bugs in your html and css files**
+
+## 🛠 Built With Ojwang Briton<a name="built-with"></a>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **It's able to detect html bugs**
+- **It's able to detect css bugs**
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://google.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+To install a text editor e.g "VS CODE"
+To install GIT
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Commands:
+
+```sh
+  cd hello microverse
+  git clone git@github.com:Eccb7/Hello.git
+```
+
+### Install
+
+Install this project with:
+
+Command:
+
+```sh
+  cd Hello
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+Command:
+
+```
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+
+
+```
+
+```
+ 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
+👤 **Ojwang Briton**
+
+- GitHub: [@Eccb7](https://github.com/Eccb7)
+- Twitter: [@briton_otieno4](https://twitter.com/briton_otieno4)
+- LinkedIn: [briton-otieno-ba6591244](https://linkedin.com/in/briton-otieno-ba6591244)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[I'll be adding more content soon]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project you can reach me through [@briton_otieno4](https://twitter.com/briton_otieno4) and share more thoughts to help improve
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+
+I would like to thank Microverse for the support I've received so far.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a></p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] Kindly add the `LICENSE` in your root directory. Make sure to click on the link after adding it to make sure it working fine as it's currently not working. Without any license, the default is that all rights are reserved. If someone other than the author wants to use this code, they have the obligation of obtaining a valid license.

Reply:
I believe now the file is present. Thanks